coops-ui/src/views/AnnualReport.vue

```diff
@@ -424,7 +422,7 @@ export default {
               // set the Initial AGM Date in the AGM Date component
               // NOTE: AR Filing Year (which is needed by agmDate component) was already set by Todo List
               this.initialAgmDate = annualReport.annualGeneralMeetingDate
-              if (annualReport.annualGeneralMeetingDate == null) {
+              if (!annualReport.annualGeneralMeetingDate) {
```

Comment:
Is `annualReport.annualGeneralMeetingDate` defined explicitly as a boolean or defined in general? Either way, this is a good refactor. It now covers null, undefined, NaN, false, or an empty string.  Good job!

Reply:
It's a string. I was going to replace == with === but I realized that == also handles other falsy cases, so I just used ! directly.